### PR TITLE
Fix event listeners for menu/copy/tweet

### PR DESCRIPTION
**Pull request by bogas04**
* Moved copy/tweet event listeners to renderShabad, where they should
reside.
* Added eventListeners to a DOMReady callback.
* Fixed a typo in code where event listeners weren't attached to element
but window object itself.

### CODE REVIEW

File: src/js/renderShabad.js [hunk 1]
```diff
@@ -91,6 +91,27 @@ function metaData(data, nav) {
...
 
 function Baani(gurbani) {
+  const onCopyClick = ({ currentTarget }) => {
+    const textarea = currentTarget.parentNode.parentNode.querySelector('textarea');
+    textarea.style.display = 'block';
+    textarea.focus();
+    textarea.select();
+    document.execCommand('copy');
+    textarea.blur();
+    textarea.style.display = 'none';
```
Comment (bogas04) [inline src/js/renderShabad.js]: Not sure if changing `style.display` is needed.

File: src/js/renderShabad.js [hunk 2]
```diff
@@ -91,6 +91,27 @@ function metaData(data, nav) {
...
+  };
+
+  const onTweetClick = ({ currentTarget}) => {
+    let tweet = currentTarget.parentNode.parentNode.querySelector('textarea').value;
+    const shortURL = `\n${shortenURL()}`;
+    if (tweet.length + shortURL.length > 134) {
+      tweet = `${tweet.substring(0, 132 - shortURL.length)}..`;
```
Comment (bogas04) [inline src/js/renderShabad.js]: @navdeepsinghkhalsa can we move this to 280 - 8 now, as twitter has increased the limit.

File: src/js/renderShabad.js (hunk 1 above)
Comment (navdeepsinghkhalsa) [reply src/js/renderShabad.js]: I believe that was put in because it otherwise didn't perform the select+copy

File: src/js/renderShabad.js (hunk 2 above)
Comment (navdeepsinghkhalsa) [reply src/js/renderShabad.js]: Yes, please. Will await that change before merging

**Comment by bogas04** [comment]
Yeah I'll have a look. The way we manage event listeners isn't the nicest way. So it's about time we actually fix it.

**Pull request merged**